webapp/channels/src/plugins/registry.ts

```diff
@@ -126,6 +126,18 @@ export default class PluginRegistry {
         return dispatchPluginComponentAction('PostMessageAttachment', this.id, component);
     });
 
+    // Register a component to the add to the post message menu.
+    // Accepts a React component. Returns a unique identifier.
+    registerPostActionComponent = reArg(['component'], ({component}: DPluginComponentProp) => {
```

Comment:
If we're not going to add this to the bottom, can we locate this "closer" to the existing `registerPostDropdownMenuAction` and clarify the difference between the two types of registrations?

Reply:
I moved that right before the DropdownMenuAction registry function and slightly changed the comments. PTAL.